src/pyggdrasil/visualize/tree.py

```diff
@@ -0,0 +1,118 @@
+"""Methods for visualizing trees."""
+
+import os
+from pathlib import Path
+
+from anytree.exporter import DotExporter
+import pydot
+import matplotlib.pyplot as plt
+from networkx.drawing.nx_pydot import graphviz_layout
+import networkx as nx
+
+
+from pyggdrasil import TreeNode
+
+
+def plot(tree: TreeNode, save_name: str, save_dir: Path, print_options: dict) -> None:
+    """Plot a tree and save it to a file.
+
+    Args:
+        tree: TreeNode
+            tree to plot, given the root node
+                tree.data: may contain info to print for tree / node e.g.
+                    "log-likelihood": float
+                        likelihood of the tree
+                tree.name: name of tree, may use a title of plot
+        save_name: str
+            name of the file to save the plot to
+        save_dir: Path
+            directory to save the plot to
+        print_options: dict
+            options for printing the tree
+                "title": bool
+                    whether to print the name/title of the tree/root
+                "data_tree": dict
+                    what attributes to print of the tree
+    Returns:
+        None
+
+    Note:
+        see tests/visualize/test_tree.py for example usage
+    """
+    # make full path
+    fullpath = os.path.join(save_dir, save_name)
+    # make output directory if it doesn't exist
+    os.makedirs(save_dir, exist_ok=True)
+
+    # convert to networkX graph
+    # convert to dot graph
+    dot_string = ""
+    for line in DotExporter(tree):
+        dot_string += line
+    # convert to networkX graph
+    graphs = pydot.graph_from_dot_data(dot_string)
```

Comment:
Is it possible to specify here node relabeling? We may want to (a) enumerate mutations starting from 1, rather than 0 or (b) use biological mutation names.

(If it's too hard for a quick fix, we can also address it later.)

Reply:
All implemented. - see the comment below.